# times: times series
# values: values matrix
float64[] times
float64[] values
